packages/cloudrun/install.mjs

```diff
@@ -0,0 +1,606 @@
+})
+	.toString()
+	.trim();
+
+execSync(
+	`echo "Project set to ${blueTextColor}${projectID}${resetTextColor}\n"`,
+	{
+		stdio: 'inherit',
+	}
+);
+
+/****************************************
+ * Check if user is configuring GCP Project, or just needs to generate .env file
+ ****************************************/
+function taskPrompt() {
+	return new Promise((resolve) => {
+		const rl = readline.createInterface({
+			input: process.stdin,
+			output: process.stdout,
+		});
+
+		rl.question(
+			`What would you like to do?\n[1] Setup the GCP project ${projectID} as a Remotion Cloud Run environment.\n[2] ${projectID} is already set up for Remotion, update to a newer Remotion version.\n[3] ${projectID} is already set up for Remotion, generate a new .env file or manage keys for the Remotion Service Account.\n${blueTextColor}`,
+			async (answer) => {
+				// reset terminal color
+				rl.output.write(`\n${resetTextColor}`);
+
+				if (answer.trim() === '1') {
+					rl.output.write(
+						`${blueTextColor}<Terraform selected>\n\n${resetTextColor}`
+					);
+					rl.close();
+					return resolve('runTerraform');
+				}
+
+				if (answer.trim() === '2') {
+					rl.output.write(
+						`${blueTextColor}<Update Remotion selected>\n\n${resetTextColor}`
+					);
+					rl.close();
+					return resolve('updateRemotion');
+				}
+
+				if (answer.trim() === '3') {
+					rl.output.write(
+						`${blueTextColor}<.env creation selected>\n\n${resetTextColor}`
+					);
+					rl.close();
+					return resolve('generateEnv');
+				}
+
+				rl.close();
+				execSync(
+					`echo "${redTextColor}Invalid selection. Please enter 1, 2 or 3.\n${resetTextColor}"`,
+					{
+						stdio: 'inherit',
+					}
+				);
+
+				const result = await taskPrompt();
+
+				resolve(result);
+			}
+		);
+	});
+}
+
+const selection = await taskPrompt();
+
+switch (selection) {
+	case 'runTerraform':
+		await runTerraform();
+		break;
+
+	case 'updateRemotion':
+		await updateRemotion();
+		break;
+
+	case 'generateEnv':
+		await generateEnv();
+		break;
+
+	default:
+		break;
+}
+
+function envPrompt() {
+	return new Promise((resolve) => {
+		const rl = readline.createInterface({
+			input: process.stdin,
+			output: process.stdout,
+		});
+
+		rl.question(
+			`\n${greenTextColor}Do you want to generate the .env file for your code base? ${blueTextColor}`,
+			async (answer) => {
+				// reset terminal color
+				rl.output.write(`${resetTextColor}`);
+				rl.close();
+
+				if (['yes', 'y'].indexOf(answer.trim().toLowerCase()) >= 0)
+					return resolve(true);
+
+				if (['no', 'n'].indexOf(answer.trim().toLowerCase()) >= 0)
+					return resolve(false);
+
+				console.log('Invalid response.');
+				const result = await envPrompt();
+
+				resolve(result);
+			}
+		);
+	});
+}
+
+function checkTerraformStateFile() {
+	/****************************************
+	 * Check for existing Terraform State
+	 ****************************************/
+
+	const dirFiles = execSync('ls -a', {
```

Comment:
this could also use `fs.readdirSync()`, probably easier to parse since it returns an array